runtime/permissions/prompter.rs

```diff
@@ -198,18 +205,22 @@ impl PermissionPrompter for TtyPrompter {
     let _stderr_guard = std::io::stderr().lock();
 
     // print to stderr so that if stdout is piped this is still displayed.
-    const OPTS: &str = "[y/n] (y = yes, allow; n = no, deny)";
-    eprint!("┌ {PERMISSION_EMOJI}  ");
+    let opts: &str = if is_unary {
+      "[y/n/a/d] (y = yes, allow; n = no, deny; a = yes to all, allow all; d = no to all, deny all)"
+    } else {
+      "[y/n] (y = yes, allow; n = no, deny)"
+    };
+    eprint!("{PERMISSION_EMOJI}  ┌ ");
```

Comment:
Your rebase seems to have reverted the change that was done in https://github.com/denoland/deno/pull/17708 where the EMOJI was moved inside the `┌` blocks.

Reply:
🥲

Reply:
Fixed!